src/ansys_sphinx_theme/search/fuse_search.py

```diff
@@ -55,9 +55,18 @@ def build_sections(self):
             section_title = node[0].astext()
             section_text = "\n".join(
                 n.astext()
-                for node_type in [nodes.paragraph, nodes.literal_block]
+                for node_type in [
```

Comment:
We need to find a better solution for this. I think projects should be able to index on a page-basis. If we allow globbing, it could be very powerful, allowing users to customize the index level on chapter-basis too.

```
index_patterns = {
    "api*": ALL_NODES,
    "examples*": PARAGRAPH,
    ...
}
```

Reply:
yeah sure, we can try to get this

Reply:
works for me